typed-builder-macro/src/field_info.rs

```diff
@@ -345,6 +378,58 @@ impl ApplyMeta for SetterSettings {
     }
 }
 
+#[derive(Debug, Clone)]
+pub enum StripOption {
```

Comment:
Nit: a better design would have been:

```rust
pub struct StripOption {
    pub fallback: Option<String>, // not sure it should even be a string, but that's for another comment
    span: Span,
}
```

That would allow for adding more sub-settings in the future, and also make it possible to use `ApplyMeta` for parsing `StripOption`.

Reply:
Nice, I've applied this.